tools/pot/openvino/tools/pot/statistics/statistic_graph_builder.py

```diff
@@ -136,23 +142,23 @@ def insert_abs_max(self, model_graph, node, type_stat, node_name, **kwargs):
         out_port = self.get_out_port(node_name)
         if out_port is not None:
             node_name = f'{node_name[0]}.{out_port}'
-        abs_node = Abs(node.graph, {"name": f'abs_{node_name}'}). \
+        clean_name = node_name.split("|")[-1]
```

Comment:
I think getting the node name this way is a little weird. Is the `node.name` not applicable here?

Reply:
As far as I remember, `node.name` is not applicable here if the node has multiple outputs.